### ISSUE TITLE
`<pf-chip-group>`: overflow or close button are inaccessible by keyboard

### ISSUE THREAD

**Issue by nikkimk** [patternfly/patternfly-elements#2708]
## Description of the issue

Using only a keyboard, the  overflow and close buttons cannot be focused and clicked.

### Impacted component(s)

- `pf-chip-group`

### Steps to reproduce

1. In docs, go to [Chip group with removable categories](https://patternflyelements.org/components/chip/#chip-group-with-removable-categories)
2. Use <kbd>Tab</kbd> to set focus on first chip in a chip group.
3. Attempt to use right arrow key <kbd>&rarr;</kbd> to navigate to **"1 more"** overflow button.
4. Attempt to use right arrow key <kbd>&rarr;</kbd> to navigate to close button.
5. Neither button can be accessed via roving tabindex.

### Expected behavior

Both the overflow or close buttons should be accessible via roving tabindex.

**Comment by bennypowers** [comment]
If this fixed in #2707 ?

**Comment by nikkimk** [comment]
> If this fixed in #2707 ?

No. While reviewing #2707 I found the issue.

**Comment by bennypowers** [comment]
Closed by #2709
